fujiji-client/src/components/ListingForm/ListingForm.jsx

```diff
@@ -206,7 +206,15 @@ export default function ListingForm({
       setUploadErrorMessage('Allowed files: .png, .jpg, .jpeg');
     } else {
       setUploadedImage(uploadedFile);
-      setListingImageUrl(URL.createObjectURL(uploadedFile));
+      const url = onImageUpload(uploadedFile);
+      url.then((result) => setListingImageUrl(result.data.imageUrl)).catch(() => {
+        toast({
+          title: 'Oops! Unable to upload the picture, We are working on it!',
+          status: 'error',
+          duration: 9000,
+          isClosable: true,
+        });
```

Comment:
for the `.then` callback we need something like this:

https://github.com/devin-efendy/fujiji/blob/2687187f4d27eea2ae7ac143ff4f3cfef1aba433/fujiji-client/src/components/ListingForm/ListingForm.jsx#L173-L199

We already catch the error from Axios in `uploadImage.js`. and we return the object that contains `error` field

Reply:
I am not sure what status code does azure function is returning right now and therefore handled the error in a different way for now and will fix it in a later sprint. Noted the issue

Reply:
The changes does not need to be the exact same like the code I linked. We just need to check if the there is a `error` field. The status code does not really matter.